app/scripts/lib/experiments/grouping-rules/token-code.js

```diff
@@ -37,6 +38,10 @@ define(function (require, exports, module) {
       if (client && this.bernoulliTrial(client.rolloutRate, subject.uniqueUserId)) {
         return this.uniformChoice(GROUPS, subject.uniqueUserId);
       }
+      const isSync = this.get('service') === 'Sync';
```

Comment:
@vbudhram are you going to be able to distinguish between `Sync` and  `ROLLOUT_CLIENTS[subject.clientId]` in the metrics data with this current setup? for example how well this feature works for sync vs amo?

Reply:
I believe so? The data posted to `/metrics` contain a service value. Is there another place I might need to specify it?

Reply:
> contain a service value

Yeap, and if we can determine that + the group of the experiment then it should be good 👍